### DISCUSSION TITLE
How to build the demo(Next.js docs) project on my machine?

### DISCUSSION THREAD

**Discussion by taozuhong** [vercel/next.js#64332] [Help]
### Summary

I am learning the Next.js framework, I want to clone and run the docs project on my machine, 
https://github.com/vercel/next.js/tree/canary/docs

How could I implement it?

**Comment by icyJoseph** [comment]
Hi,

The actual docs rendering, and the site code are closed source.

**Comment by taozuhong** [comment]
I feel sad with it, We need a productive repo to learn Nextjs, the docs repo is the best choice for it.
